docs/user/setup.rst

```diff
@@ -86,14 +78,14 @@ installation of Arch Linux.
 .. code-block:: shell
 
     $ sudo dnf install llvm clang
-    $ sudo dnf install libunwind-devel gc-devel zlib-devel re2-devel # optional
+    $ sudo dnf install gc-devel zlib-devel # both optional
```

Comment:
You can split this in two lines and then say something like:
```
$ sudo dnf install gc-devel # optional, only if you use boehm gc
$ sudo dnf install gc-devel # optional, only if you use java.util.zip.*
```
IMO this makes it more clear what are these dependencies for. 

It would be good to update the comment in this style everywhere for optional deps in setup instructions.

Reply:
Lets change this later in another PR, it's not related to re2s.